.editorconfig

```diff
@@ -0,0 +1,10 @@
+# http://editorconfig.org
```

Comment:
I was using emacs in evil-mode (vim implemented in emacs lisp) for a long time, but recently tried out spacemacs instead. But lately I picked up atom, and while I miss modal-editing it's been really good so far :-) All of those are supported by editor config btw :-)